src/services/legal-services.ts

```diff
@@ -281,4 +281,20 @@ export default class LegalServices {
       return data.business
     })
   }
+
+  /**
+   * Fetches an extraprovincial company's business info from COLIN.
+   * @param businessId the business identifier
+   * @returns a promise to return the business info, else exception
+   */
+  static async fetchColinBusinessInfo (businessId: string): Promise<BusinessIF> {
+    // FUTURE: update this as needed
+    const url = `businesses/${businessId}?entityType=A`
+
+    return axios.get(url).then(response => {
+      const data = response?.data
+      if (!data) throw new Error('Invalid API response')
+      return data.business
+    })
+  }
```

Comment:
Initial version. At a minimum, the URL will have to be updated. And maybe this will need to be moved to a new services file (since it's changing from a Legal API call to a COLIN call).

This will be done in bcgov/entity#21138.